test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -38,6 +38,13 @@
 #ifdef _WIN32
 #include "os_windows.hpp"
 #endif
+#ifdef AIX
+# include <sys/mman.h>
+// sys/mman.h defines MAP_ANON_64K beginning with AIX7.3 TL1
+# ifndef MAP_ANON_64K
+# define MAP_ANON_64K  0x400
+# endif
+#endif
```

Comment:
Could you move the AIX-specific files out to something like test_os_aix.cpp, please? (same directory, just use ifdef AIX in the whole file). Its nicer, and precedence is there with the Cgroup tests of linux.

Reply:
I hope I did it like you meant. Please verify.

Reply:
Yes you're right, I had tested this out beforehand, but was reluctant and not sure if I change something not obvious with this.